index.html

```diff
@@ -7,17 +7,41 @@
 </head>
 
 <body>
-    <div id="architecture-list">
-    </div>
+    <!-- BEGIN CONTENT SECTION -->
+    <div id="architecture-list"></div>
```

Comment:
can we add a padding-bottom: 10px; to this div? The label for adding more architectures gets a bit overhung